src/lib.rs

```diff
@@ -28,26 +28,32 @@
 //! - `std`: Enables functionality that depends on the standard library. This
 //!   is a superset of `alloc` and adds interoperation with standard library types
 //!   and traits.
-//! - `clock`: Enables reading the system time (`now`) that depends on the standard library for
-//! UNIX-like operating systems and the Windows API (`winapi`) for Windows.
+//! - `clock`: Enables reading the local timezone (`Local`). This is a superset of `now`.
+//! - `now`: Enables reading the system time (`now`).
 //! - `wasmbind`: Interface with the JS Date API for the `wasm32` target.
 //!
 //! Optional features:
 //!
-//! - [`serde`][]: Enable serialization/deserialization via serde.
-//! - `rkyv`: Enable serialization/deserialization via rkyv.
+//! - `serde`: Enable serialization/deserialization via [serde].
+//! - `rkyv`: Enable serialization/deserialization via [rkyv]. This is equivalent to `rkyv-32`.
```

Comment:
We should probably discourage the use of this feature in the documentation, we only have it for compatibility. The replacement features where introduced in #1368.
No preference from my side on how to word it. Maybe just "`rkyv`: Deprecated, use the `kkyv-*` features"?

Reply:
Sounds good to me!